packages/docs-app/src/examples/core-examples/breadcrumbsExample.tsx

```diff
@@ -48,13 +48,13 @@ const ITEMS: BreadcrumbProps[] = [
     { icon: "folder-close", text: "All files" },
     { icon: "folder-close", text: "Users" },
     { icon: "folder-close", text: "Janet" },
-    { href: "#", icon: "folder-close", text: "Photos" },
-    { href: "#", icon: "folder-close", text: "Wednesday" },
+    { href: "#core/components/overflow-list", icon: "folder-close", text: "Photos" },
```

Comment:
actually, this component is used at both `#core/components/overflow-list` and `#core/components/breadcrumbs`. so we'll need a different solution that links to the current page in each usage of `<BreadcrumbsExample>`.